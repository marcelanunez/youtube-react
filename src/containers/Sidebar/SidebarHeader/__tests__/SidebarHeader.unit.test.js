import React from 'react';
import { shallow } from 'enzyme';
import { SidebarHeader } from '../SideBarHeader';

describe('SidebarHeader', () => {
  test('renders SidebarHeader with props.title=null', () => {
    const wrapper = shallow(<SidebarHeader />);
    expect(wrapper).toMatchSnapshot();
  });

  test("renders SidebarHeader with props.title=''", () => {
    const wrapper = shallow(<SidebarHeader title="" />);
    expect(wrapper).toMatchSnapshot();
  });

  test('renders SidebarHeader with sample title', () => {
    const wrapper = shallow(<SidebarHeader title="Sample Title" />);
    expect(wrapper).toMatchSnapshot();
  });
});
