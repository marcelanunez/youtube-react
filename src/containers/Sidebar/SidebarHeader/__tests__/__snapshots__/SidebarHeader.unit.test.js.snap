// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SidebarHeader renders SidebarHeader with props.title='' 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  />
</MenuItem>
`;

exports[`SidebarHeader renders SidebarHeader with props.title=null 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  />
</MenuItem>
`;

exports[`SidebarHeader renders SidebarHeader with sample title 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  >
    SAMPLE TITLE
  </MenuHeader>
</MenuItem>
`;
